src/test/java/org/apache/commons/collections4/MapUtilsTest.java

```diff
@@ -159,6 +159,17 @@ public void testInvertMap() {
         assertEquals( "3", out.get("C"));
         assertEquals( "4", out.get("D"));
         assertEquals( "5", out.get("E"));
+
```

Comment:
I would put this in a new method `testInvertMapNull` or something like that, this will give us better granularity in test code.

Reply:
+1, @dota17 other than that this change looks good. If you could update it whenever you have time I think it should be ready to be merged.

Reply:
Thank you for your review,I had fixed it.
Any others?

Reply:
Thank you for your review. I had fixed it and resolved the conflict.